.github/workflows/codeql-analysis.yml

```diff
@@ -12,11 +12,7 @@
 name: 'CodeQL'
 
 on:
-  push:
-    branches: [master]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [master]
+  workflow_dispatch:
```

Comment:
Are we running the CodeQL not on PRs and master anymore?

Reply:
Fixed